src/common/transformations/src/transformations/utils/utils.cpp

```diff
@@ -456,6 +456,20 @@ bool is_on_constant_path(const ov::Output<ov::Node>& output) {
     return status;
 }
 
+bool process_subgraph(ov::pass::ModelPass& model_pass, const std::shared_ptr<Node>& node) {
+    bool changed = false;
+
+    if (const auto& multi_subgraph_op = std::dynamic_pointer_cast<op::util::MultiSubGraphOp>(node)) {
+        for (const auto& sub_graph : multi_subgraph_op->get_functions()) {
+            if (sub_graph) {
+                changed |= model_pass.run_on_model(sub_graph);
```

Comment:
```suggestion
                changed = model_pass.run_on_model(sub_graph) || changed;
```
As this is logical operation not bitwise.